tests/data/make_metricvalue_butlers.py

```diff
@@ -33,8 +33,16 @@ def add_metricvalues(butler, plus):
     adding ``plus`` to the values that are stored (to allow different repos to
     have different Measurement values).
     """
-    dimensions = {"instrument", "visit", "detector"}
     storageClass = "MetricValue"
+
+    # a metric that only depends on instrument
+    dimensions = {"instrument"}
+    addDatasetType(butler, "metricvalue_verify_instrumentTest", dimensions, storageClass)
+    value = Measurement("verify.instrumentTest", (2 + plus)*u.dimensionless_unscaled)
+    butler.put(value, "metricvalue_verify_instrumentTest", {"instrument": "TestCam"}, run=collection)
```

Comment:
Does this commit break the build? Does it need to be merged with f52b928?

Reply:
I don't think there was ever an expectation that each commit be able to build on its own? These make complete sense as separate commits to me: "new test data", "thing that fixes tests".

Reply:
The [developer guide](https://developer.lsst.io/work/flow.html#git-commit-organization-best-practices) says commits should be "self-contained". That's generally understood (in the industry, if not in Project) to mean "does not break the build", since otherwise you'd be in trouble if you rearranged/reverted only one of the two commits, or tried to run `git bisect` in a range that included the non-self-contained commits.